source/util/LanIp.py

```diff
@@ -0,0 +1,26 @@
+import os
+import socket
+
+if os.name != "nt":
+    import fcntl
+    import struct
+    def get_interface_ip(ifname):
+        s = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
```

Comment:
Got this code directory from stack overflow while trying to get it to work. Said to myself that I would clean it up after everything was working. As expected I forgot it.